Sources/TestHelper/BillingServiceMock.swift

```diff
@@ -30,6 +30,7 @@ public final class BillingServiceMock: BillingService {
     private var _sessionStates: [BillingHandler.SessionType: BillingHandler.SessionState] = [:]
     private var _events: [Event] = []
 
+    public var accessToken: String = ""
```

Comment:
Should we use some non-empty string here and cover it with a test?

Reply:
slightly different though